docs/RFCs/011-Sdk-Testing-Platform.md

```diff
@@ -0,0 +1,84 @@
+# RFC 011 - Run dotnet test with Microsoft.Testing.Platform
+
+- [x] Approved in principle
+- [x] Under discussion
+- [ ] Implementation
+- [ ] Shipped
+
+## Current State
+
+Currently, when we run `dotnet test` in CLI, we use vstest as a test runner/driver to run tests in test projects.
+
+## Motivation
+
+With `dotnet test`, users should be able to use [Microsoft testing platform](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-intro?tabs=dotnetcli#microsofttestingplatform-pillars) to run their tests for the sake of improving their experience. They should have the option to opt-in/out this new experience.
+
+The reason for opting-in/out this experience is 
+1. if we were to auto-infer if test projects were using vstest or the testing platform is we will end up with many vulnerabilities, thus making it hard to handle this correctly.
```

Comment:
this sentence is complicated, and hard to parse. I am not even sure if I am following the spirit of it below:

```suggestion
1. autodetecting if test projects are using vstest or the testing platform is complex, and we would end up with many false positives, making it hard to deliver a working and consistent experience.
```